src/components/common/select/Select.scss

```
@@ -1,12 +1,14 @@
+@use '../../../assets/sass/variables/colors';
+
 .select {
     position: relative;
-    width: 179px;
-    height: 40px;
+    width: 11.1875rem;
+    height: 2.5rem;
     display: flex;
-    padding-left: 16px;
-    padding-right: 16px;
+    padding-left: 1rem;
+    padding-right: 1rem;
```

This can be simplified as follows:


```suggestion
    padding: 0 1rem;
```

---

Your suggestion was implemented! Thank you for your time